otherlibs/str/str.mli

```diff
@@ -50,21 +50,20 @@ val regexp : string -> regexp
    - [\     ] Quotes special characters.  The special characters
               are [$^\.*+?[]].
 
-   Note: the argument to [regexp] is usually a string literal. In this
-   case, any backslash character in the regular expression must be
-   doubled to make it past the OCaml string parser. For example, the
-   following expression:
-   {[ let r = Str.regexp "hello \\([A-Za-z]+\\)" in
-      Str.replace_first r "\\1" "hello world" ]}
+   In regular expressions you will often use backslash characters; it's
+   easier to use a verbatim string literal [{|...|}] to avoid having to
```

Comment:
The OCaml manual calls them "quoted string literals", not "verbatim string literals". ("quoted" is maybe confusing here, but let's use the standard name anyway.)